app/views/candidate_interface/shared/_task_list.html.erb

```diff
@@ -40,13 +40,24 @@
         path: application_form_presenter.other_qualification_path,
       )) %>
     </li>
-    <li class="app-task-list__item">
-      <%= render(TaskListItemComponent.new(
-        text: t('page_titles.degree'),
-        completed: application_form_presenter.degrees_completed?,
-        path: application_form_presenter.degrees_path,
-      )) %>
-    </li>
+
+    <% if application_form_presenter.no_degree_and_degree_not_completed? && teacher_degree_apprenticeship_feature_active? %>
+      <li class="app-task-list__item">
+        <%= render(TaskListItemComponent.new(
+            text: t('page_titles.degree'),
+            completed: application_form_presenter.degrees_completed?,
+            path: candidate_interface_degree_university_degree_path,
+          )) %>
+      </li>
+    <% else %>
+      <li class="app-task-list__item">
+        <%= render(TaskListItemComponent.new(
+          text: t('page_titles.degree'),
+          completed: application_form_presenter.degrees_completed?,
+          path: application_form_presenter.degrees_path,
+        )) %>
+      </li>
+    <% end %>
```

Comment:
The only difference here is the path -- can we not update the `degrees_path` method to return the correct path for the first condition as well? Then we don't have to add more conditionals to this view.

Reply:
Done on ed64a20c01